src/main/java/com/conveyal/gtfs/error/NewGTFSErrorType.java

```diff
@@ -24,18 +24,18 @@ public enum NewGTFSErrorType {
     DUPLICATE_TRIP(Priority.MEDIUM, "More than one trip had an identical schedule and stops."),
     FARE_TRANSFER_MISMATCH(Priority.MEDIUM, "A fare that does not permit transfers has a non-zero transfer duration."),
     FEED_TRAVEL_TIMES_ROUNDED(Priority.LOW, "All travel times in the feed are rounded to the minute, which may cause unexpected results in routing applications where travel times are zero."),
-    FLEX_FORBIDDEN_ARRIVAL_TIME(Priority.HIGH, "It is forbidden to define an arrival time when either start pickup drop off window or end pickup drop off window are defined."),
-    FLEX_FORBIDDEN_DEPARTURE_TIME(Priority.HIGH, "It is forbidden to define a departure time when either start pickup drop off window or end pickup drop off window are defined."),
-    FLEX_FORBIDDEN_DROP_OFF_TYPE(Priority.HIGH, "It is forbidden to define a drop off type of 0 (Regularly scheduled pickup) when a stop id refers to a stop area or location."),
-    FLEX_FORBIDDEN_STOP_AREA_AREA_ID(Priority.HIGH, "It is forbidden to define a stop area area id with the same value as a stop id or location id."),
-    FLEX_FORBIDDEN_LOCATION_ID(Priority.HIGH, "It is forbidden to define a location id with the same value as a stop id."),
-    FLEX_FORBIDDEN_MEAN_DURATION_FACTOR(Priority.HIGH, "It is forbidden to define a mean duration factor when a stop id does not refer to a stop area or location."),
-    FLEX_FORBIDDEN_MEAN_DURATION_OFFSET(Priority.HIGH, "It is forbidden to define a mean duration offset when a stop id does not refer to a stop area or location."),
-    FLEX_FORBIDDEN_SAFE_DURATION_FACTOR(Priority.HIGH, "It is forbidden to define a safe duration factor when a stop id does not refer to a stop area or location."),
-    FLEX_FORBIDDEN_SAFE_DURATION_OFFSET(Priority.HIGH, "It is forbidden to define a safe duration offset when a stop id does not refer to a stop area or location."),
+    FLEX_FORBIDDEN_ARRIVAL_TIME(Priority.HIGH, "It is forbidden to define an arrival time when either start pickup drop off window or end pickup drop off windows are defined."),
```

Comment:
Why is the second window plural? Also, is it "is defined" (singular)? (see e.g. https://preply.com/en/blog/using-either-neither-nor-and-or-in-english/) I think the other ones below should be singular too. Also, would it be better to use the exact field names?